lib/core-data/components.js

```diff
@@ -78,10 +79,18 @@ function assertCamelCasedProps(item, name) {
  * @return {object}
  */
 export function getSchema(uri, path, testStore) {
-  const name = isComponent(uri) ? getComponentName(uri) : uri,
-    prefix = `state.schemas['${name}']`;
+  let schema, result, name, prefix;
 
-  let schema, result;
+  if (isComponent(uri)) {
+    name = getComponentName(uri);
+    prefix = `state.schemas['${name}']`;
+  } else if (isLayout(uri)) {
+    name = getLayoutName(uri);
+    prefix = 'state.layout.schema';
+  } else {
```

Comment:
actually, looking at the code again, it looks like this function supports being called with a _component name_ or a uri, so going to keep the third branch just in case that functionality is used anywhere